packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -367,6 +367,43 @@ export class QuoteRequestor {
         return expirationTimeMs.isLessThan(currentTimeMs.plus(this._expiryBufferMs));
     }
 
+    /**
+     * Gets both standard RFQ makers and "alternative" RFQ makers and combines them together
+     * in a single configuration map. If an integration key whitelist is present, it will be used
+     * to filter a specific makers.
+     *
+     * @param options the RfqmRequestOptions passed in
+     * @param assetOfferings the RFQM or RFQT maker offerings
+     * @returns a list of TypedMakerUrl instances
+     */
+    public static getTypedMakerUrls(
```

Comment:
hmm, this function is doing two things - I overlooked the second part when reading it the first time:

1. Combines regular and alt asset offerings into one typed array
2. Filters out maker urls unless they are approved

I would either rename it to something like `getTypedMakerUrlsAndWhitelist` or break into two functions: `getTypedMakerUrls` and `selectWhitelistedMakerUrls`